packages/frontend/src/lib/table/model/ModelColumnActions.svelte

```diff
@@ -17,8 +18,20 @@ async function goDeleteModel() {
   await studioClient.deleteLocalModel(object.id);
   deleteConfirmVisible= false;
 }
+
+function openModelFolder() {
+  console.log(object.file);
+  if (object && object.file) {
+    studioClient.openURL('file://'+object.file.path);
```

Comment:
I would use uris everywhere

build the URI using
```
Uri.file(object.file.path)
```

and then not have in studioClient
```
  async openURL(url: string): Promise<boolean> {
    return await podmanDesktopApi.env.openExternal(podmanDesktopApi.Uri.parse(url));
  }
```
but just accept URI (to avoid double parse/unexpected behavior)

Reply:
I did not pay enough attention to backend/frontend usage so you may disregard my comment as basically frontend has no access to Podman Desktop API and the URI class.

so either you revert the commit or you have `openFile(filePath: string)` method on studio-client that will do `openExternal(URI.file(filePath))`

sorry for the noise